templates/inventory/inventory.js

```diff
@@ -25,22 +37,18 @@ async function fetchRequiredPageData() {
 
 async function createPriceCell(homeDetails) {
   const { price } = homeDetails;
-  const numericPrice = price ? parseFloat(price.replace(/[^\d.-]/g, '')) : null;
-
```

Comment:
This was required to get only the numbers from the price, removing any special characters like commas or symbols.

Reply:
No need for this as the data coming in is raw data without the formatting.